array-methods.js

```diff
@@ -65,7 +105,23 @@ var sumOfBankBalances = null;
   take each `amount` and add 18.9% interest to it rounded to the nearest cent
   and then sum it all up into one value saved to `sumOfInterests`
  */
+
 var sumOfInterests = null;
+let total = 0;
+
+function findStates(arr) {
+  if (arr.state === 'WI' || arr.state === 'IL' || arr.state === 'OH' || arr.state === 'GA' || arr.state === 'WY' || arr.state === 'DE') {
```

Comment:
Try and see if you can use an array and the array methods to find a shorter solution, compared to listing out all the possible if conditions.